python_configure.bzl

```diff
@@ -441,3 +443,62 @@ Args:
       will configure for that Python version instead of the installed
       binary. This configuration takes precedence over python_version.
 """
+
+def _parse_my_own_version_from_module_dot_bazel(ctx):
```

Comment:
I believe you can get the root module version from `modules.version` if `modules.is_root` is true.

https://bazel.build/rules/lib/builtins/bazel_module.html

Reply:
Oh, this is just pybind11_bazel parsing its own version so that it can fetch the same version of pybind11. In general use, it's not going to be the root module.

Reply:
Hmm, I think it's still better to define the actual pybind11 version separately, maybe via an additional module extension tag? Because what if you want to update pybind11_bazel version but still use an old version of pybind11?

Reply:
My mental model is that the versions should be tightly coupled because, **_ideally_**, pybind11_bazel wouldn't be separate in the first place, but I suppose decoupling them would require less toil by @rwgk after this Bzlmod support stabilises? OTOH, `compatibility_level` seems like the only mechanism for avoiding conflicts, so I'm paranoid about losing that?

Reply:
OK, SG. I guess we can still decouple later if that's really necessary.